main.tf

```diff
@@ -20,7 +29,7 @@ data "aws_secretsmanager_secret" "ssh_key" {
 }
 
 resource "aws_cloudformation_stack" "this" {
-  name               = var.name
+  name               = "${var.name}-${uuid()}"
```

Comment:
Isn't there a more shelf stable way to do this? You could SHA on the template body and achieve the same thing maybe?

Reply:
Since I'm ignoring changes to `name`, the name will only change if a replacement is needed, making this more stable than a sha on the template body or anything else I can think of. This prevents accidental collisions and allows for create_before_destroy to work.